docs/book/payments.rst

```diff
@@ -4,112 +4,136 @@
 Payments
 ========
 
-Sylius contains a very flexible payments management system with support for many gateways. (payment providers)
-We are using very powerful payment abstraction library, called [Payum], which handles all sorts of capturing, refunding and recurring payments logic.
+Sylius contains a very flexible payments management system with support for many gateways (payment providers).
+We are using a payment abstraction library - `Payum <https://github.com/Payum/Payum>`_, which handles all sorts of capturing, refunding and recurring payments logic.
 
 On Sylius side, we integrate it into our checkout and manage all the payment data.
 
 Payment
 -------
 
-Every payment in Sylius, successful or failed, is represented by *Payment* model, which contains basic information and reference to appropriate order.
+Every payment in Sylius, successful or failed, is represented by the **Payment** model, which contains basic information and a reference to appropriate order.
 
-Payment has following attributes:
+Payment State Machine
+---------------------
 
-* id
-* currency (code)
-* amount
-* reference to *PaymentMethod*
-* reference to *Order*
-* state
-* reference to [PaymentSource] (optional)
-* createdAt
-* updatedAt
+A Payment that is assigned to an order will have it's own state machine with a few available states:
+``cart``, ``new``, ``processing``, ``completed``, ``failed``, ``cancelled``, ``void``, ``refunded``.
 
-All these properties are easily accessible through simple API:
+The available transitions between these states are:
 
-.. code-block:: php
+.. code-block:: yaml
 
-    <?php
+    transitions:
```

Comment:
https://github.com/MadMind/StateMachineVisualizationBundle could be worth a try.